SU-SWS/decanter
# ARCH-184 added images element and all necessary files to apply image defaults

# READY FOR REVIEW

# Summary
- Set defaults for images (margins, padding, height, width, and display), to help with responsiveness and layout

# Needed By (Date)
- As soon as possible (would unblock some project work)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
1. Ensure that the image element is displaying in the style guide and doing what added mixin suggests that it should

# Affected Projects or Products
- decanter
- stanford_basic (theme)
- suhumsci (theme), Archaeology project

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-184
- https://github.com/SU-SWS/decanter/pull/175

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
